go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.9.0
 	github.com/Azure/go-autorest/autorest/date v0.2.0
 	github.com/google/uuid v1.3.0
-	github.com/microsoft/moc v0.11.0-alpha.11
+	github.com/microsoft/moc v0.11.0-alpha.11-peidlu
```

Comment:
Hold this PR until https://github.com/microsoft/moc/pull/185 is merged.

Reply:
You can use `moc@v0.11.0-alpha.12` now.